src/aleph/vm/models.py

```diff
@@ -236,7 +239,7 @@ def create(
                 )
             elif self.hypervisor == HypervisorType.qemu:
                 assert isinstance(self.resources, AlephQemuResources)
-                self.vm = vm = AlephQemuInstance(
+                self.vm = vm = AlephQemuConfidentialInstance(
```

Comment:
These should be fixed to avoid using always a confidential class. You can use the property `is_confidential` to make the filter.